src/components/Anchor/styles/sharedStyles.js

```diff
@@ -25,7 +25,7 @@ export const base = css`
   height: auto;
   margin: auto;
 
-  &:focus {
+  ${({ disabled }) => (disabled ? 'pointer-events: none' : '')} &:focus {
```

Comment:
Do we need a `;` between these two statements? Then they can be on separate lines.

Reply:
Probably Prettier doing that. It's syntactically ok, but yeah, I wish they'd improve the parsing on CSS.